apphub/semantic_segmentation/unet3d/3DUnetMRBrainSegmentation_Isensee.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import os
+import tempfile
+
+import nibabel as nib
+import tensorflow as tf
+
+import fastestimator as fe
+from fastestimator.architecture import UNet3D_Isensee
+from fastestimator.dataset import brats
+from fastestimator.op import NumpyOp, TensorOp
+from fastestimator.op.tensorop import Loss, ModelOp
+from fastestimator.op.tensorop.augmentation import Augmentation3D
+from fastestimator.trace import LRController, ModelSaver
+
+
+class NIBImageReader(NumpyOp):
```

Comment:
are the Ops here general enough that we ought to move them to the main code base?

Reply:
moved it to main folder.